Tests/MapboxNavigationTests/NavigationViewControllerTests.swift

```diff
@@ -295,17 +296,19 @@ class NavigationViewControllerTests: TestCase {
         service.locationManager!(service.locationManager, didUpdateLocations: [fultonStreetLocation])
     }
     
-    func testDestinationAnnotationUpdatesUponReroute() {
+    func disabled_testDestinationAnnotationUpdatesUponReroute() {
```

Comment:
This test doesn't fail locally. Do we need to disable it?

Reply:
Let's document disabled tests in different ticket then so that we can prioritize fixes in future.

Reply:
I opened a ticket in #3861 to track the disabled tests with the reasons.